tutorials/2_Gazebo_and_ROS.ipynb

```diff
@@ -0,0 +1,1330 @@
+    "    <joint name=\"${name}_joint\" type=\"fixed\">\n",
+    "      <xacro:insert_block name=\"origin\" />\n",
+    "      <parent link=\"${parent}\"/>\n",
+    "      <child link=\"${name}_link\"/>\n",
+    "    </joint>\n",
+    "\n",
+    "    <link name=\"${name}_link\">\n",
+    "      <inertial>\n",
+    "        <mass value=\"0.001\" />\n",
+    "        <origin xyz=\"0 0 0\" rpy=\"0 0 0\" />\n",
+    "        <inertia ixx=\"0.000000017\" ixy=\"0\" ixz=\"0\" iyy=\"0.000000017\" iyz=\"0\" izz=\"0.000000017\" />\n",
+    "      </inertial>\n",
+    "      <visual>\n",
+    "        <origin xyz=\"0 0 0\" rpy=\"0 0 0\" />\n",
+    "        <geometry>\n",
+    "          <box size=\"0.01 0.01 0.01\" /> \n",
+    "        </geometry>\n",
+    "      </visual>\n",
+    "      <collision>\n",
+    "        <origin xyz=\"0 0 0\" rpy=\"0 0 0\" />\n",
+    "        <geometry>\n",
+    "          <box size=\"0.01 0.01 0.01\" />\n",
+    "        </geometry>\n",
+    "      </collision>\n",
+    "    </link>\n",
+    "\n",
+    "    <gazebo reference=\"${name}_link\">\n",
+    "      <sensor type=\"ray\" name=\"${name}\">\n",
+    "        <always_on>true</always_on>\n",
+    "        <update_rate>${update_rate}</update_rate>\n",
+    "        <pose>0 0 0 0 0 0</pose>\n",
+    "        <visualize>false</visualize>\n",
+    "        <ray>\n",
+    "          <scan>\n",
+    "            <horizontal>\n",
+    "              <samples>${ray_count}</samples>\n",
+    "              <resolution>1</resolution>\n",
+    "              <min_angle>-${field_of_view/2}</min_angle>\n",
+    "              <max_angle> ${field_of_view/2}</max_angle>\n",
+    "            </horizontal>\n",
+    "            <vertical>\n",
+    "              <samples>${ray_count}</samples>\n",
+    "              <resolution>1</resolution>\n",
+    "              <min_angle>-${field_of_view/2}</min_angle>\n",
+    "              <max_angle> ${field_of_view/2}</max_angle>\n",
+    "            </vertical>\n",
+    "          </scan>\n",
+    "          <range>\n",
+    "            <min>${min_range}</min>\n",
+    "            <max>${max_range}</max>\n",
+    "            <resolution>0.01</resolution>\n",
+    "          </range>\n",
+    "        </ray>\n",
+    "\n",
+    "        <plugin name=\"gazebo_ros_${name}_controller\" filename=\"libhector_gazebo_ros_sonar.so\">\n",
+    "          <gaussianNoise>0.005</gaussianNoise>\n",
+    "          <topicName>${ros_topic}</topicName>\n",
+    "          <frameId>${name}_link</frameId>\n",
+    "        </plugin>\n",
+    "      </sensor>\n",
+    "    </gazebo>\n",
+    "  </xacro:macro>\n",
+    "</robot>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p>Plugin tag refers to the plugins of Gazebo used to generate sensory feedback from the environment. We will use hector gazebo plugins for simulating sonar sensor. If you dont have, you can install using the command</p>"
```

Comment:
**A** plugin tag ...

Remove installation instructions as it will be installed on the VM.